### ISSUE TITLE
ClientConnectionException: Invalid response received: 401; invalid JWT

### ISSUE THREAD

**Issue by b-liberman** [Consensys-Incorporated/teku#6160]
### Description
After an upgrade to 22.9.0 I started getting the following exception:

2022-09-06 12:26:01.444+00:00 | pool-3-thread-3 | ERROR | TimeBasedEth1HeadTracker | Failed to update eth1 chain head
org.web3j.protocol.exceptions.ClientConnectionException: Invalid response received: 401; invalid JWT

at org.web3j.protocol.http.HttpService.performIO(HttpService.java:178) ~[core-4.9.3.jar:?]
at org.web3j.protocol.Service.send(Service.java:48) ~[core-4.9.3.jar:?]
at org.web3j.protocol.Service.lambda$sendAsync$0(Service.java:60) ~[core-4.9.3.jar:?]
at org.web3j.utils.Async.lambda$run$1(Async.java:38) ~[core-4.9.3.jar:?]
at java.util.concurrent.CompletableFuture$AsyncRun.run(CompletableFuture.java:1800) ~[?:?]
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130) ~[?:?]
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630) ~[?:?]
at java.lang.Thread.run(Thread.java:832) ~[?:?]

### Versions (Add all that apply)
* Software version: 22.9.0
* Java version: 14
* OS Name & Version: Ubuntu, 20.04.2 LTS (Focal Fossa)
* Docker Version: --
* Cloud VM, type, size: AWS 5.xlarge

Thanks a lot!

**Comment by ajsutton** [comment]
This is probably going to be easier to help with via discord (https://discord.gg/7hPv2T6) but this indicates that the execution engine rejected the JWT authentication. Most likely the execution client has a different JWT secret to Teku.

**Comment by b-liberman** [comment]
I went to discord. Hope to get some help. However, this is maybe helpful for the analysis?

2022-09-06 12:59:15.797+00:00 | pool-3-thread-3 | WARN  | Web3jEth1Provider | Endpoint https://mainnet.infura.io/v3/72c... is INVALID | Invalid response received: 401; invalid JWT

I configured 

--ee-endpoint=https://mainnet.infura.io/v3/72c... 

Is this not correct? It used to work with previous versions...
